components/search/SearchInterface.vue

```diff
@@ -398,9 +401,17 @@
           qf: this.qf,
           query: this.query,
           reusability: this.reusability,
-          view: this.view
+          view: this.view,
+          theme: this.theme,
+          api: this.api
         };
 
+        // override when current theme is newspaper and the new theme is not
+        // TODO: this is a bit of a hack, find a better way
+        if (this.theme === 'newspaper' && updates.theme !== 'newspaper') {
+          current.api = null;
```

Comment:
This prevents the api parameter being preserved within the newspapers collection. Remove it, and apply the following change to the `queryUpdatesForFacetChanges` function:
```diff
diff --git a/components/search/SearchInterface.vue b/components/search/SearchInterface.vue
index 3b6321d..1a6f7cc 100644
--- a/components/search/SearchInterface.vue
+++ b/components/search/SearchInterface.vue
@@ -356,7 +356,7 @@
           // Remove collection-specific filters when collection is changed
           // TODO: this is rather crude and will not scale well; improve
           if (name === 'THEME' && !selected['api']) {
-            delete filters['api'];
+            filters['api'] = null;
           }
         }
```